LondonBritishLibrary/orient/BLorient9798.xml

```diff
@@ -0,0 +1,281 @@
+                            <locus from="58va" to="61vb"/>
+                            <title>Amharic explanation of the Creed</title>
+                            <note>Amharic explanation of the <ref type="work" corresp="#LIT3308Nicene">Creed</ref> followed by the Gǝʿǝz 
+                                text with a phrase-by-phrase Amharic translation.</note>
+                            <incipit xml:lang="am">ጸሎተ፡ ሃይማኖት፡ የተናገሩበት፡ ምክንያት፡ አነጋገራቸው፡ ምነው፡ ቢሉ፡ አርዮስ፡ የሚሉ፡ ዲያቆን፡ ነበረ፡ ወልድን፡ ፍጡር፡ አለ፡
+                                ይህን፡ ግን፡ ዘመኑ፡ በዲዮቅልጥያኖስ፡ ነው፡</incipit>
+                            <textLang mainLang="am" otherLangs="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">61</measure>
+                                    <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                                        <height>200</height>
+                                        <width>170</width>
+                                        <depth/>
+                                    </dimensions>
+                                </extent>
+                                <condition key="good">Stains on several folia. Holes in the parchment have been patched.</condition>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="21"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Neat handwriting, possibly several hands.</desc>
+                                <date notBefore="1700" notAfter="1868" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus target="#56v"/>
+                                <desc>Notes in a Ḥasaba ḥǝgg, which notes the death of Emperor Tewodros and must be therefore in the year 
+                                    <date when-custom="521" calendar="grace" when="1868">521</date> or shortly after.</desc>
+                                <date when="1868">1868</date>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#49rb"/>
+                                    <desc type="GuestText">Calculation of the auspicious and ill-omened days 
```

Comment:
"GuestText" in additional notes is encoded in a slightly different way. I think you have to structure it like, for instance: 
`<desc type="GuestText"><title type="incomplete" ref="LIT1104Anapho"/>An excerpt from the Anaphora of the Apostles</desc>` (here and below)

Reply:
I changed it for a1, but for a2, I have no LIT and no information to identify the work.
```

<item xml:id="a1">
                                    <locus target="#49rb"/>
                                    <desc type="GuestText"><title type="incomplete" ref="LIT4145Onausp"/>Calculation of the auspicious and 
                                        ill-omened days</desc>
                                    <q xml:lang="gez"><title>ሐሳበ፡ ተአምር፡ ጽንዕት፡ ወጽንጽንት፡</title> በመስከረም፡ ፲ወ፪ጽንዕት። ወላዕሉ፡ ጽንፅንት። በጥቅምት፡ ፳፫ጽንፅንት፡ 
                                        ወላዕሉ፡ ጽንዕት፡</q>
                                </item>
                                <item xml:id="a2">
                                    <locus target="#49v"/>
                                    <desc type="GuestText">A fragment of an Amharic work concerning the Trinity.</desc>
                                </item>
```